build_docs

```diff
@@ -237,6 +246,11 @@ def run_build_docs(args):
             docker_args.extend(['--publish', '8000:8000/tcp'])
             docker_args.extend(['--publish', '8001:8001/tcp'])
             open_browser = OpenBrowserState()
+
+            # ulimit has to get bumped to allow the container to do the
+            # more-expensive polling-style Parcel watcher on Docker for Mac;
+            if environ.get("CHOKIDAR_USEPOLLING"):
```

Comment:
I feel like it'd be a little cleaner to check for mac here and add the environment variable as a `-e` to docker, also right here. *Technically* you only need this if you have both `--doc` and `--open` but it won't hurt anything to add it if you just have `--open`.

Reply:
good call!